independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/core/CodestartSpec.java

```diff
@@ -85,20 +85,23 @@ public boolean isPreselected() {
         private final Map<String, Object> data;
         private final Map<String, Object> sharedData;
         private final List<CodestartDep> dependencies;
+        private final List<CodestartDep> boms;
         private final List<CodestartDep> testDependencies;
 
         public LanguageSpec() {
-            this(null, null, null, null);
+            this(null, null, null, null, null);
         }
 
         @JsonCreator
         public LanguageSpec(@JsonProperty("data") Map<String, Object> data,
                 @JsonProperty("shared-data") Map<String, Object> sharedData,
                 @JsonProperty("dependencies") List<CodestartDep> dependencies,
+                @JsonProperty("boms") List<CodestartDep> boms,
```

Comment:
Do you see a case were a bom could be declared by a codestart? if not I would drop this, the using the inputData should be enough..